machine_spraak_streaming/lambda/message/main.py

```diff
@@ -0,0 +1,49 @@
+import boto3
+import os
+import json
+import base64
+import io
+from scipy.io.wavfile import read, write
```

Comment:
Suggestion: `read` and `write` are quite ambiguous function names. I would suggest importing them as `wav_read` and `wav_write`, or just import `scipy.io.wavfile` and use `wavfile.read` and `wavfile.write`.

Reply:
Very good comment, will change this 👍